scripts/set_comment_modified_default_false.py

```diff
@@ -0,0 +1,44 @@
+"""
+For unmodified comments, change comment.modified from None to False since the default value
+has been set to False on the comment model.
+"""
+
+import sys
+import logging
+from modularodm import Q
+
+from framework.transactions.context import TokuTransaction
+from scripts import utils as script_utils
+from website.app import init_app
+from website.project.model import Comment
+
+logger = logging.getLogger(__name__)
+
+
+def main(dry=True):
+    init_app(routes=False)
+    with TokuTransaction():
+        do_migration(get_targets())
+        if dry:
+            raise Exception('Abort Transaction - Dry Run')
+
+
+def get_targets():
+    return Comment.find(Q('modified', 'eq', None))
+
+
+def do_migration(records):
+    count = 0
```

Comment:
Minor: if you're modifying every record without any conditionals, you could just use `len(records)` rather than maintaining this `count` variable.